testing/matrix-sdk-integration-testing/src/tests/sliding_sync/room.rs

```diff
@@ -353,6 +358,7 @@ impl UpdateObserver {
     }
 }
 
+/*
```

Comment:
Why wouldn't we instead [ignore](https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html#ignoring-tests) those test? That way I don't need to search for commented out tests and I can still try to run them with the `--ignored` flag.

Reply:
Because they are `tokio::test` and I didn't find a way to ignore a tokio test :-/.

Reply:
Did you try with `#[ignore]`? `tokio::test` is just sugar to create a runtime and run the test inside the runtime: https://docs.rs/tokio-macros/latest/tokio_macros/attr.test.html#using-the-multi-thread-runtime.

Reply:
There, it will add the `#[test]` for you: https://github.com/tokio-rs/tokio/blob/82628b8a7828507617a430e6875f1e25d5c211ef/tokio-macros/src/entry.rs#L432C30-L435.

Reply:
Interesting. Thanks for the tips! Did not know it was falling back to the std proc-macro.